### PR TITLE
Update locale pt-br

### PR DESCRIPTION
**Pull request by helldr**
Added:
dayNames,
dayNamesShort,
monthNames,
monthNamesShort

### CODE REVIEW

File: locale/pt-br.js [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
 		week: "Semana",
 		day: "Dia",
-		list: "Compromissos"
+		list: "Lista"
 	},
 	allDayText: "dia inteiro",
 	eventLimitText: function(n) {
 		return "mais +" + n;
 	},
+	dayNames:['Domingo','Segunda','Terça','Quarta','Quinta','Sexta','Sábado'],
```
Comment (arshaw) [inline locale/pt-br.js]: are the month translations that momentjs offers not satisfactory?
https://github.com/moment/moment/blob/2.18.1/locale/pt-br.js#L14-L17

Comment (FilipePintoReis) [reply locale/pt-br.js]: @arshaw They are. Despite this, today and list translations are correct. Previous list translation means Tasks

**Comment by arshaw** [comment]
in v4, the translations now happen with the browser's Intl API, but we'll still want to have the updated text for `list`. Will merge just that in the next release.

**Comment by arshaw** [comment]
this is released in v4.4.0:
https://github.com/fullcalendar/fullcalendar/releases/tag/v4.4.0

**Pull request merged**